classes/functions.php

```diff
@@ -371,4 +354,16 @@ public function issueCheck($series_id, $issue_number) {
       $this->issueExists = 0;
     }
   }
+
+  public function collectionCount($user_id) {
+    $this->db_connection = new mysqli ( DB_HOST, DB_USER, DB_PASS, DB_NAME );
+    if ($this->db_connection->connect_errno) {
+      die ( "Connection failed:" );
+    }
+
+    $sql = "SELECT user_id, comic_id, quantity
+        FROM users_comics
+        WHERE user_id = $user_id";
+    $this->total_issue_count = mysqli_num_rows($this->db_connection->query ( $sql ));
+  }
```

Comment:
This counts the total number of comics that exists in their collection. Series count will also eventually go here.